### PR TITLE
feat: Support other5-other10 IDs

### PR DESCRIPTION
**Pull request by rmi22186**
We need to support other5-other10 ids.

### CODE REVIEW

File: src/Amplitude.js [hunk 1]
```diff
@@ -145,6 +151,15 @@ var constructor = function () {
 
     function onUserIdentified(user) {
         var userIdMissing;
+
+        function setUserId(userId) {
+            if (userId) {
+                return getInstance().setUserId(userId);
```
Comment (alexs-mparticle) [inline src/Amplitude.js:157]: @rmi22186  What is the expected return of this method?

File: src/Amplitude.js [hunk 2]
```diff
@@ -145,6 +151,15 @@ var constructor = function () {
...
+            } else {
+                userIdMissing = true;
```
Comment (alexs-mparticle) [inline src/Amplitude.js:159]: Out of curiosity, instead checking for a `userIdMissing` boolean, why not just check to see if `userId` is `undefined` or `null`?

Comment (rmi22186) [reply src/Amplitude.js]: good call.  this can be simplified considerably

File: src/Amplitude.js (hunk 1 above)
Comment (rmi22186) [reply src/Amplitude.js]: i'll simplify this.  seems like it's not necessary to return given `onUserIdentified` is a `void` function and doesn't return anything

**Review by alexs-mparticle — APPROVED** [review]

**Pull request merged**